src/algebraic_geometry/prime_spectrum.lean

```diff
@@ -415,6 +402,9 @@ def basic_open (r : R) : topological_space.opens (prime_spectrum R) :=
 { val := { x | r ∉ x.as_ideal },
   property := ⟨{r}, set.ext $ λ x, set.singleton_subset_iff.trans $ not_not.symm⟩ }
 
+lemma is_open_basic_open {a : R} : is_open ((basic_open a).val) :=
```

Comment:
```suggestion
lemma is_open_basic_open {a : R} : is_open ((basic_open a) : set (prime_spectrum R)) :=
```
I think there is a coercion from `opens` to `set`, so let's use that. (Unless I'm wrong)